src/styles/blocks/product.scss

```diff
@@ -0,0 +1,79 @@
+.product {
+  box-sizing: border-box;
+  display: flex;
+  flex-direction: column;
+  padding: 32px 16px 16px;
+  width: 200px;
+  background: #fff;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+}
+
+.product__image {
+  height: 134px;
+  width: 160px;
+  margin: 0 auto 40px;
+}
+
+.product__title {
+  margin: 0 0 4px;
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 18px;
+  color: $color-main-accent;
+}
+
+.product__code {
+  margin: 0 0 16px;
+  font-weight: 400;
+  font-size: 10px;
+  line-height: 14px;
+  color: $color-secondary;
+}
+
+.product__rating {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  margin-bottom: 24px;
+  font-weight: 400px;
```

Comment:
Change 'font-weight: 400px;' to 'font-weight: 400;' as font-weight doesn't require a unit.